### PR TITLE
Mass conversion from ABAP system to files

### PR DESCRIPTION
**Pull request by albertmink**
details see the commits

### CODE REVIEW

File: file-formats/chkv/type/zif_aff_chkv_v1.intf.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
-  "$schema": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/chkv/chkv.json",
+  "$schema": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/intf/intf.json",
```
Comment (larshp) [inline file-formats/chkv/type/zif_aff_chkv_v1.intf.json:2]: this looks wrong

File: file-formats/clas/type/zif_aff_clas_v1.intf.abap [hunk 2]
```diff
@@ -74,9 +74,18 @@ INTERFACE zif_aff_clas_v1 PUBLIC.
...
+      types                TYPE zif_oo_aff_types_v1=>ty_component_descriptions,
+      "! <p class="shorttext">Attribute Descriptions</p>
+      "! Attribute descriptions
+      attributes           TYPE zif_oo_aff_types_v1=>ty_component_descriptions,
+      "! <p class="shorttext">Event Descriptions</p>
+      "! Event descriptions
+      events               TYPE zif_oo_aff_types_v1=>ty_events,
+      "! <p class="shorttext">Method Descriptions</p>
+      "! Method descriptions
+      methods              TYPE zif_oo_aff_types_v1=>ty_methods,
```
Comment (larshp) [inline file-formats/clas/type/zif_aff_clas_v1.intf.abap:88]: looks like this reverts the changes from https://github.com/SAP/abap-file-formats/pull/124

Comment (albertmink) [reply file-formats/clas/type/zif_aff_clas_v1.intf.abap:88]: thanks, that has to be changed

**Comment by albertmink** [comment]
fb4e7c13e09e403af612f94eaaf6dd0ad376f036 has to be reverted

**Comment by albertmink** [comment]
This branch/pr is used as a vehicle to show and discuss the diffs between A: what the report writes and B: content of git repo.

File: file-formats/chkv/type/zif_aff_chkv_v1.intf.json (hunk 1 above)
Comment (huber-nicolas) [reply file-formats/chkv/type/zif_aff_chkv_v1.intf.json:2]: yes, this is a mistake.Thank you for finding it. I will correct it

Comment (huber-nicolas) [reply file-formats/chkv/type/zif_aff_chkv_v1.intf.json:2]: "$schema": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/intf/intf.json"
is correct. This is a serialized interface and therefore intf/intf is correct.
It is the interface which contains the abap type of chkv objects.

I also always stumble over this. It's annoying :-)

**Comment by albertmink** [comment]
cherry-picked commits in #220

**Comment by albertmink** [comment]
continued in #222

**Pull request closed without merging**